src/services/storage/handlers/stake.ts

```diff
@@ -11,13 +12,13 @@ const handlers = {
   async Staked (event: EventData, { stakeService }: StorageServices): Promise<void> {
     const { user: account, total, token, amount } = event.returnValues
 
-    const stake = await StakeModel.findOne({ where: { token, account } })
+    let stake = await StakeModel.findOne({ where: { token, account } })
```

Comment:
Maybe a cleaner solution here would be [`findOrCreate`](https://sequelize.org/v5/class/lib/model.js~Model.html#static-method-findOrCreate)?

Reply:
Good idea